lib/modules/manager/tekton/extract.ts

```diff
@@ -80,6 +85,74 @@ function getDeps(doc: TektonResource): PackageDependency[] {
   return deps;
 }
 
+const taskAnnotation = regEx(/^pipelinesascode\.tekton\.dev\/task(-[0-9]+)?$/);
+
+function addPipelineAsCodeAnnotations(
+  annotations: Record<string, string> | undefined | null,
+  deps: PackageDependency[],
+): void {
+  if (is.nullOrUndefined(annotations)) {
+    return;
+  }
+
+  for (const [key, value] of Object.entries(annotations)) {
+    if (!taskAnnotation.test(key)) {
+      continue;
+    }
+
+    const tasks = value.replace('/]$/', '').replace('/^[/', '').split(',');
```

Comment:
```suggestion
    const tasks = value.replace(regEx('/]$/'), '').replace(regEx('/^[/'), '').split(',');
```

Otherwise it would only do string replace 😉